Formula/sbt.rb

```diff
@@ -2,6 +2,7 @@ class Sbt < Formula
   desc "Build tool for Scala projects"
   homepage "https://www.scala-sbt.org/"
   url "https://piccolo.link/sbt-1.2.4.tgz"
+  mirror "http://tiny.ph/lXhU"
```

Comment:
Can we use HTTPS for this?

Reply:
yeah, Missed it actually it works as HTTPS also. Will do.

Reply:
I can't reach `tiny.ph` here…

Reply:
Added it

Reply:
We are using it for collecting stats, I am planning to add github release link as one more minor.
Does that make sense?

Reply:
As this URL isn't versioned it won't be simple to update, please use the GitHub link instead of this one.